src/main/java/uk/gov/hmcts/befta/dse/ccd/DataLoaderToDefinitionStore.java

```diff
@@ -168,9 +179,113 @@ protected boolean shouldTolerateDataSetupFailure() {
     @Override
     protected void doLoadTestData() {
         addCcdRoles();
+        createRoleAssignments();
         importDefinitions();
     }
 
+    public void createRoleAssignments() {
+        getRoleAssignmentFiles(RA_DATA_RESOURCE_PACKAGES);
+    }
+
+    private void getRoleAssignmentFiles(String[] resourcePackages) {
+        try {
+            ClassPath cp = ClassPath.from(Thread.currentThread().getContextClassLoader());
+            for (String resourcePackage : resourcePackages) {
+                String prefix = resourcePackage + "/";
+                for (ClassPath.ResourceInfo info : cp.getResources()) {
+                    if (info.getResourceName().startsWith(prefix)
+                            && info.getResourceName().endsWith(".ras.json")) {
+                        File resource = new ClassPathResource(info.getResourceName()).getFile();
+                        createRoleAssignment(resource);
+                    }
+                }
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    protected void createRoleAssignment(File resource) {
+        try {
+            String payload = new String(Files.readAllBytes(resource.toPath()));
+            JSONObject payLoadJSONObject = new JSONObject(payload);
+            Response response = asRoleAssignmentUser().given()
+                    .header("Content-type", "application/json")
+                    .body(readObjectFromJsonFile(payLoadJSONObject).toString())
```

Comment:
I see `body()` also accepts `File` as an argument too - just wondering if this would work rather than needing to convert to a `JSONObject` only to write back as a String.
```suggestion
                    .body(resource)
```
If that doesn't work, shouldn't this?
```suggestion
                    .body(payload)
```

Reply:
throws exception as
Caused by: java.lang.RuntimeException: Calling Role Assignment service failed with response body: {
    "errorCode": 400,
    "errorMessage": "Bad Request",
    "errorDescription": "The input parameter: \"[[$ROLE_ASSIGNMENT_SOLICITOR_USER]]\", does not comply with the required pattern",
    "timeStamp": "22-09-2021 08:26:05.036"
}